### PR TITLE
🚧  Add Splashpage and associated logo 

### PR DESCRIPTION
**Pull request by fgottesmann**
I'm currently working on the Splashpage which is almost finished, but I need to find a method to set something like a box-shadow for the svg.

### CODE REVIEW

**Review by Dev-dfm — APPROVED** [review]
lgtm 💯

**Review by MaxMuecket — APPROVED** [review]
lgtm 🚀

nice BEM

**Review by dhnazarenko — APPROVED** [review]
LGTM 🚀

File: src/app/pages/Splashpage/Splashpage.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import styles from './Splashpage.module.css';
+
+type SplashpageProps = {
+  imgSrc: string;
+};
+
+function Splashpage({ imgSrc }: SplashpageProps): JSX.Element {
+  return (
+    <div className={styles.splash}>
+      <main>
```
Comment (lmachens) [inline src/app/pages/Splashpage/Splashpage.tsx:11]: You can remove the outer div if you like
```suggestion
    <main className={styles.splash}>
```

**Review by lmachens — COMMENTED** [review]
good job

File: src/app/pages/Splashpage/Splashpage.tsx (hunk 1 above)
Comment (fgottesmann) [reply src/app/pages/Splashpage/Splashpage.tsx:11]: You're right. I removed it. Ty!

Comment (lmachens) [reply src/app/pages/Splashpage/Splashpage.tsx:11]: @fgottesmann :),  you should remove it in this pull request before merging next time

**Pull request merged**